getting_started/workflow/export/exporting_for_android.rst

```diff
@@ -55,8 +65,11 @@ Scroll down to the section where the Android settings are located:
 In that screen, the path to 3 files needs to be set:
 
 -  The *adb* executable (adb.exe on Windows)
+  - This will usually be found at ```%appdata%/../local/Android/Sdk/platform-tools/adb.exe```
```

Comment:
```suggestion
  - It can usually be found at ``%LOCALAPPDATA%\Android\Sdk\platform-tools\adb.exe``.
```

Reply:
looks good.   sorry for all the grammar / style errors :(